samples/SkiaAndroidSample/settings.gradle.kts

```diff
@@ -6,4 +6,7 @@ pluginManagement {
     }
 
 }
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
+}
```

Comment:
I'd prefer to avoid adding new dependencies unless strictly necessary.
Building Skiko already requires a sophisticated environment setup described in the DEVELOPMENT.md, so I believe it is enough to just update the instruction there with the new JDK version.

Reply:
Yeah, thanks, I know the plugin. Just wanted to avoid adding unnecessary things to the build.